webcompat/static/js/lib/bugform.js

```diff
@@ -285,99 +293,106 @@ BugForm.prototype.enableSubmits = function() {
   this.submitButtons.removeClass("is-disabled");
 };
 
+/* determines function based on whether validation returns true or false */
+BugForm.prototype.determineValidityFunction = function(func, field, silent) {
+  if (func(field)) return "makeValid";
```

Comment:
nit: can we do multi-line conditional here? (or did prettier do this for you?)

Reply:
added